haiku-games/connect4/connect4-1.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Haiku clone of Connect-4"
+DESCRIPTION="Connect-4 is a game where players take turns dropping pieces \
+down a column. The first player to get four consecutive pieces in a row, \
+column, or diagonal wins the game."
+HOMEPAGE="https://github.com/HaikuArchives/Connect4/"
```

Comment:
You can drop the trailing slash here because the URI is still valid without it.
```diff
-HOMEPAGE="https://github.com/HaikuArchives/Connect4/"
+HOMEPAGE="https://github.com/HaikuArchives/Connect4"
```
(The "rules" are explained in [`HOMEPAGE`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#homepage).)